AddSerilog require a receiver ILoggingBuilder

I'm following github pages to get started with serilog; but visual studio show me the error in subject.

Seems that .AddSerilog() is not available for interface ILoggerFactory, but it is for ILoggingBuilder

I'm building in .NET Core 2

a snippet of what i'm doing
```
  public static void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
        {
            var applicationLifetime = app.ApplicationServices.GetRequiredService<IApplicationLifetime>();
            applicationLifetime.ApplicationStopping.Register(OnShutdown);
            
            loggerFactory.AddSerilog();
        }
```

Getting the same error in serilog.extensions.logging 2.0.0, .Net Core 2 Preview 2

```
error CS1929: 'ILoggerFactory' does not contain a definition for 'AddSerilog' and the best extension method overload 'SerilogLoggingBuilderExtensions.AddSerilog(ILoggingBuilder, ILogger, bool)' requires a receiver of type 'ILoggingBuilder'
```

--- a/src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj
+++ b/src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj
@@ -50,7 +50,6 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0-preview2-final" />
     <PackageReference Include="NETStandard.Library" Version="2.0.0-preview2-25401-01" />
-    <Compile Remove="SerilogLoggerFactoryExtensions.cs" />
   </ItemGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net46' ">

